### PR TITLE
Feature/remember users after login

### PR DESCRIPTION
**Pull request by robcole**
Adds support for remembering users after logging into slack. Defaults to 1 week 
cookie expiration.

All comments appreciated!

### CODE REVIEW

File: spec/features/authentication_spec.rb [hunk 1]
```diff
@@ -14,6 +16,11 @@
           )
         )
       end
+
+      scenario 'changes the login text' do
+        expect(page).not_to have_content('Sign In With Slack')
```
Comment (fozcodes) [inline spec/features/authentication_spec.rb]: I think we should be using i18n here. That's my bad - I should have done that when I put in that navbar item.

File: spec/features/authentication_spec.rb [hunk 2]
```diff
@@ -14,6 +16,11 @@
...
+        expect(page).to have_content('Sign Out')
```
Comment (fozcodes) [inline spec/features/authentication_spec.rb]: Same here - i18n

File: app/views/layouts/_navigation_links.html.erb [hunk 3]
```diff
@@ -1,3 +1,7 @@
 <li class="">
+<% if current_user %>
+  <%= link_to "Sign Out", logout_path, class: 'navbar-brand' %>
+<% else %>
   <%= link_to 'Sign In With Slack', login_path('slack'), class: 'navbar-brand' %>
```
Comment (fozcodes) [inline app/views/layouts/_navigation_links.html.erb]: If we use i18n for these, we won't have to worry about changing the wording and breaking tests.

File: app/views/layouts/_navigation_links.html.erb [hunk 4]
```diff
@@ -1,3 +1,7 @@
...
+<% end %>
```
Comment (fozcodes) [inline app/views/layouts/_navigation_links.html.erb]: Could we move this conditional into a view helper?

**Comment by fozcodes** [comment]
Hey Rob, I think this looks good. A couple of small comments from me. Another thing I think we need to do overall is get off the Controller tests and on to Integration tests, but that's probably something we can refactor after we get this thing out into the world.

Let me know what you think!

**Comment by robcole** [comment]
@fosberyi - Made the recommended changes, everything made sense. I'm not sure about controller spec vs. integration specs when we're testing cookies (happy to revisit that down the road) but beyond that, here you go.

**Comment by fozcodes** [comment]
@robcole sweet looks good man! Merge away!

**Comment by robcole** [comment]
I don't have merge permissions - feel free to merge or let me know what to do next in the event I'm missing something here.

**Pull request merged**